configs/openmrs/initializer_config/ampathforms/NCD_Screening.json

```diff
@@ -111,7 +111,7 @@
                             "id": "hxHeartDisease",
                             "questionOptions": {
                                 "rendering": "radio",
-                                "concept": "7cf6d87c-4a23-467d-83c1-cd5efacd8cd1",
```

Comment:
@kazlaw is there an issue arising from using the existing concept uuid?

Reply:
Hi @kdaud by editing the existing concepts we would have conflicts since the concepts similar are already present so I just edited the existing ones since they are the same 

![image](https://github.com/openmrs/ozone-distro-cambodia/assets/36139599/fe7dc035-9b99-468c-8134-13801bb5ffed)